_posts/2015-03-16-CocoaPods.org-Take-Two.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title:  "CocoaPods.org Take Two"
+author: orta
+categories: cocoapods.org releases website
+---
+
+TL;DR: [CocoaPods.org][1] version 2.1 has been released, with support for inline deep search results, and pod pages.
+
+<!-- more -->
+
+<style>
+  /* for the CP preview */
+.container > .row > .content > center > a > img {
+  border: 1px solid black;
+}
+</style>
+
+## Backstory.
+
+The CocoaPods web site is an interesting beast. In October 2013, we set around [rewriting][2] the CocoaPods web pages. We centralized the documentation, created a blog and redesigned everything. 
```

Comment:
i have never heard "set around" before, though it makes sense in context. maybe that's a british thing?